civihr_default_theme/scss/base/_custom.scss

```diff
@@ -1178,6 +1178,11 @@ form {
       margin-bottom: 0;
       vertical-align: middle;
     }
+
```

Comment:
This file indentation, line spacing, comments styling (comments should be capitalised) are wrong. Could you please fix it in this PR?